### DISCUSSION TITLE
Localizations Status

### DISCUSSION THREAD

**Discussion by holndonner** [mattwright324/youtube-metadata#9] [General]
hello!
can you please explain, how to set a video's localization to "Localizations The video does not have localizations."?
i cant get that status

**Comment by mattwright324** [comment]
Hey @holndonner,

Video localizations can be modified assuming you are the owner/publisher of the video. The message `The video does not have localizations.` will appear when no localizations are set. Localizations are set via the Subtitles section of the YouTube Studio.

See here for more details
- https://support.google.com/youtube/answer/6300772?hl=en
- https://support.google.com/youtube/answer/6289575

If you want to remove the Subtitles/localizations to get this status, I would assume you can do it from this same place where you would add them.
